consumer/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/oppgavebehandling/RestOppgaveBehandlingServiceImpl.kt

```diff
@@ -83,30 +93,76 @@ class RestOppgaveBehandlingServiceImpl @Autowired constructor(
         if (aktorId == null || aktorId.isEmpty()) {
             throw Exception("AktørId-mangler på person")
         }
-        val response = apiClient.opprettOppgave(
+        val skjermetOppgaveRequest = PostOppgaveRequestJsonDTO(
+                aktoerId = aktorId,
+                opprettetAvEnhetsnr = request.opprettetavenhetsnummer,
+                behandlesAvApplikasjon = "FS22",
+                beskrivelse = request.beskrivelse,
+                temagruppe = "",
+                tema = request.tema,
+                tildeltEnhetsnr = "",
+                tilordnetRessurs = "",
+                behandlingstema = behandling.map(Behandling::getBehandlingstema).orElse(""),
+                oppgavetype = oppgaveTypeMapped,
+                behandlingstype = behandling.map(Behandling::getBehandlingstype).orElse(""),
+                aktivDato = LocalDate.now(),
+                fristFerdigstillelse = request.oppgaveFrist,
+                prioritet = PostOppgaveRequestJsonDTO.Prioritet.valueOf(stripTemakode(request.prioritet))
+        )
+
+        val skjermpetOppgaveResponse = apiClient.opprettOppgave(
                 authorization = stsService.systemUserToken,
                 xminusCorrelationMinusID = correlationId(),
-                postOppgaveRequestJsonDTO = PostOppgaveRequestJsonDTO(
-                        aktoerId = aktorId,
-                        opprettetAvEnhetsnr = request.opprettetavenhetsnummer,
-                        behandlesAvApplikasjon = "FS22",
-                        beskrivelse = request.beskrivelse,
-                        temagruppe = "",
-                        tema = request.tema,
-                        tildeltEnhetsnr = "",
-                        tilordnetRessurs = "",
-                        behandlingstema = behandling.map(Behandling::getBehandlingstema).orElse(""),
-                        oppgavetype = oppgaveTypeMapped,
-                        behandlingstype = behandling.map(Behandling::getBehandlingstype).orElse(""),
-                        aktivDato = LocalDate.now(),
-                        fristFerdigstillelse = request.oppgaveFrist,
-                        prioritet = PostOppgaveRequestJsonDTO.Prioritet.valueOf(stripTemakode(request.prioritet))
-                )
+                postOppgaveRequestJsonDTO = skjermetOppgaveRequest
         )
 
+        val response = gjorSporring<PostOppgaveRequestJsonDTO, PostOppgaveResponseJsonDTO>(skjermetOppgaveRequest.aktoerId.toString(), skjermetOppgaveRequest, skjermpetOppgaveResponse)
+
         return OpprettOppgaveResponse(response.id?.toString() ?: throw java.lang.RuntimeException("No oppgaveId found"))
     }
 
+    private fun <S, T> gjorSporring(ident: String, request: S, response: T): T {
```

Comment:
Denne funksjonen er tenkt å kun logge? isåfall kanskje endre navn til noe som har med logging å gjøre?